<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QWidget" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,40%);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>478</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="styleSheet">
           <string notr="true">background-color:transparent</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="styleSheet">
                <string notr="true">background-color:#79634E;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
                  </property>
                  <property name="text">
                   <string>NUEVO CLIENTE</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>421</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="styleSheet">
                <string notr="true">background-color:white;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="styleSheet">
                     <string notr="true">background-color:#79634E;
min-width:6em;
max-width:6em;
min-height:1.5em;
max-height:1.5em;
padding:5px;
color:white;
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
                    </property>
                    <property name="text">
                     <string>CLIENTE</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="cliente">
                    <property name="styleSheet">
                     <string notr="true">background-color:#F2F2F2;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="styleSheet">
                     <string notr="true">background-color:#79634E;
min-width:6em;
max-width:6em;
min-height:1.5em;
max-height:1.5em;
padding:5px;
color:white;
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
                    </property>
                    <property name="text">
                     <string>REGIONAL</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="sucursal">
                    <property name="styleSheet">
                     <string notr="true">background-color:#F2F2F2;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;
</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="styleSheet">
              <string notr="true">background-color:white;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: #294D39;
padding:10px;
min-width:5em;</string>
                  </property>
                  <property name="text">
                   <string>CONTACTO</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="styleSheet">
              <string notr="true">background-color:white;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="styleSheet">
                   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: #294D39;
padding:10px;
min-width:5em;</string>
                  </property>
                  <property name="text">
                   <string>Nombre</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nombre">
                  <property name="styleSheet">
                   <string notr="true">background-color:#F2F2F2;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="styleSheet">
                   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: #294D39;
padding:10px;
min-width:5em;</string>
                  </property>
                  <property name="text">
                   <string>Teléfono</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="telefono">
                  <property name="styleSheet">
                   <string notr="true">background-color:#F2F2F2;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="styleSheet">
                   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: #294D39;
padding:10px;
min-width:5em;</string>
                  </property>
                  <property name="text">
                   <string>E-mail</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="mail">
                  <property name="styleSheet">
                   <string notr="true">background-color:#F2F2F2;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="styleSheet">
                   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: #294D39;
padding:10px;
min-width:5em;</string>
                  </property>
                  <property name="text">
                   <string>Cargo</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="cargo">
                  <property name="styleSheet">
                   <string notr="true">background-color:#F2F2F2;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:0px solid;
min-height:1.5em;
max-height:1.5em;
color: #294D39;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:white;
background-color:rgba(121,99,78,100%);;
min-width:8em;
max-width:8em;
min-height:2em;
max-height:2em;</string>
               </property>
               <property name="text">
                <string>Cancelar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="tarifario_butt">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:white;
background-color:rgba(121,99,78,100%);;
min-width:8em;
max-width:8em;
min-height:2em;
max-height:2em;</string>
               </property>
               <property name="text">
                <string>Guardar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
